StreamChatClient_v3/Database/DTOs/ChannelDTO_Tests.swift

```diff
@@ -270,29 +297,64 @@ class ChannelModelDTO_Tests: XCTestCase {
             //      Assert.willBeEqual(payload.members[0].user.unreadMessagesCount, loadedChannel?.members.first?.unreadMessagesCount)
+            
+            // Messages
+            Assert.willBeEqual(payload.messages[0].id, loadedChannel?.latestMessages.first?.id)
+            Assert.willBeEqual(payload.messages[0].type.rawValue, loadedChannel?.latestMessages.first?.type.rawValue)
+            Assert.willBeEqual(payload.messages[0].text, loadedChannel?.latestMessages.first?.text)
+            Assert.willBeEqual(payload.messages[0].updated, loadedChannel?.latestMessages.first?.updatedDate)
+            Assert.willBeEqual(payload.messages[0].created, loadedChannel?.latestMessages.first?.createdDate)
+            Assert.willBeEqual(payload.messages[0].deleted, loadedChannel?.latestMessages.first?.deletedDate)
+            Assert.willBeEqual(payload.messages[0].args, loadedChannel?.latestMessages.first?.args)
+            Assert.willBeEqual(payload.messages[0].command, loadedChannel?.latestMessages.first?.command)
+            Assert.willBeEqual(payload.messages[0].extraData, loadedChannel?.latestMessages.first?.extraData)
+            Assert.willBeEqual(payload.messages[0].isSilent, loadedChannel?.latestMessages.first?.isSilent)
+            Assert.willBeEqual(payload.messages[0].mentionedUsers.count, loadedChannel?.latestMessages.first?.mentionedUsers.count)
+            Assert.willBeEqual(payload.messages[0].parentId, loadedChannel?.latestMessages.first?.parentId)
+            Assert.willBeEqual(payload.messages[0].reactionScores, loadedChannel?.latestMessages.first?.reactionScores)
+            Assert.willBeEqual(Int32(payload.messages[0].replyCount), loadedChannel?.latestMessages.first?.replyCount)
+            
+            // Message user
+            Assert.willBeEqual(payload.messages[0].user.id, loadedChannel?.latestMessages.first?.user.id)
+            Assert.willBeEqual(payload.messages[0].user.created, loadedChannel?.latestMessages.first?.user.userCreatedDate)
+            Assert.willBeEqual(payload.messages[0].user.updated, loadedChannel?.latestMessages.first?.user.userUpdatedDate)
+            Assert.willBeEqual(payload.messages[0].user.lastActiveDate, loadedChannel?.latestMessages.first?.user.lastActiveDate)
+            Assert.willBeEqual(payload.messages[0].user.isOnline, loadedChannel?.latestMessages.first?.user.isOnline)
+            Assert.willBeEqual(payload.messages[0].user.isBanned, loadedChannel?.latestMessages.first?.user.isBanned)
+            Assert.willBeEqual(payload.messages[0].user.roleRawValue, loadedChannel?.latestMessages.first?.user.userRole.rawValue)
+            Assert.willBeEqual(payload.messages[0].user.teams, loadedChannel?.latestMessages.first?.user.teams)
         }
     }
 }
 
 extension XCTestCase {
-    func dummyPayload(with channelId: ChannelId) -> ChannelPayload<DefaultDataTypes> {
+    var dummyUser: UserPayload<NameAndImageExtraData> {
```

Comment:
Nit: To make this more useful, it's suggest passing `userId` inside as a parameter (like we do it for `cid`). It allows for easier asserting in tests.

Reply:
Didn't add it intentionally, since it's not needed yet